tests/integration/testers/coordinator/src/coordinator_config.py

```diff
@@ -32,7 +32,7 @@ def all_nonmeta():
 
 
 ROLES_CONFIG = ClusterRolesConfig(
-    roles=set(role for role in Role),
+    roles=set(*Role),
```

Comment:
do you need the star? 

<img width="843" height="542" alt="image" src="https://github.com/user-attachments/assets/7d87a932-fd58-4e77-89d0-3762c93e0f2c" />

Reply:
good catch that's my mistake